common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -99,3 +99,18 @@
 else:
     tmpl.render_context(context)
 %></%def>
+
+<%def name="optional_include_mako_with_fallback(file, fallback, with_microsite=False, **kwargs)"><%
```

Comment:
Why not just fold this functionality into the `optional_include_mako` function that already exists? If the file isn't found, check and see if a fallback was specified. If so, try to use it; if not, pass.

Also, what happens if the template author wants to define a list of fallbacks?